### ISSUE TITLE
Incorrect output for multiple modules

### ISSUE THREAD

**Issue by fazibear** [bryanjos/elixirscript#141]
Input:
```elixir
defmodule Test do
end

defmodule T do
end
```

Output:
```js
    import * as Elixir from 'Elixir';    import * as Elixir from 'Elixir';
    const __MODULE__ = Elixir.Kernel.SpecialForms.atom('Elixir.T');
    export {};    import * as Elixir from 'Elixir';
    const __MODULE__ = Elixir.Kernel.SpecialForms.atom('Elixir.Test');
    export {};
```

Output is not correct. There are multiple imports and `__MODULE__` const assingments.

**Comment by bryanjos** [comment]
When output is sent to standard out, it currently sends all of the modules with it. I added delimiters to help with splitting up the output. `//:ENDFILE` is used to specify the end of a file. The file name is also added to the output via `//<file>:ENDFILENAME` where `<file>` is the would be name of the file.

In JavaScript, the output can be split using the following code:

```javascript
      stdout.split("//:ENDFILE").forEach(function(code){
        var filenameAndCode = code.split(":ENDFILENAME");

        //the filename
        var filename = filenameAndCode[0].trim().substr(2);

        //the code
        var code = filenameAndCode[1];
      });

```

**Comment by bryanjos** [comment]
Closing this. This will be corrected properly as part of #151
